.github/CODEOWNERS

```
@@ -49,6 +49,7 @@ airflow-core/src/airflow/ui/public/i18n/locales/zh-TW/ @Lee-W @jason810496 # + @
 airflow-core/src/airflow/ui/public/i18n/locales/fr/ @pierrejeambrun @vincbeck
 airflow-core/src/airflow/ui/public/i18n/locales/tr/ @bugraoz93 # +@hasancatalgol
 airflow-core/src/airflow/ui/public/i18n/locales/hi/ @vatsrahul1001
+airflow-core/src/airflow/ui/public/i18n/locales/ca/ @jscheffl # + @ecodina
```

@bugraoz93 "<Press Commit Button here to join being translation owner>" as a redundant peer.
```suggestion
airflow-core/src/airflow/ui/public/i18n/locales/ca/ @jscheffl @bugraoz93 # + @ecodina
```

---

I cannot press commit :( It is outdated but will add new comment :)

---

Made the changes in another comment https://github.com/apache/airflow/pull/55013#discussion_r2312596566, thanks Jens!